defaults/main.yml

```diff
@@ -4,7 +4,7 @@ stack_user: stack
 stack_group: stack 
 stack_passwd: redhat
 sudoer_perm: 0440
-req_rpm: python-tripleoclient,git,mariadb-server,mariadb
-start_db: mariadb
+req_rpm: python-tripleoclient,gitr,hosp-director-images,rhosp-director-images-ipa
```

Comment:
I think that the 'r' and the ',' need to change places so that it reads `git, rhosp-director-images`